docs/user-guide/quick-start/admin-gui.md

```diff
@@ -5,78 +5,189 @@ As an Admin, you have the ability to create and deactivate users, change users'
+***Hover over the permissions to view the its description**. For example, hovering over Manage groups shows that the permission allows the user:  Access and manage of all groups*
 
-## Edit and Update Permissions
+Find more information about users and permissions, and their capabilities in the Defining User Permissions and Groups in the ODM section.
 
-* Click the icon to be redirected to the **Users and Permissions** section.
+### Edit User status
 
-![Go to Dash](quick-start-images/go-to-dashboard.png)
+As an administrator, you have the ability to manage users' permissions, statuses (active or inactive), and passwords. 
 
-* Click **Users and Permissions**.
+To edit a user's status, follow these steps:
 
-![Users and Permissions](quick-start-images/user-permissions-tab.png)
+**1. Edit User Status:**
+- To change a user's status to active or inactive, click on the three dots on the right side of the user's name.
+- Select **Deactivate** to turn the user inactive immediately. To reactivate a deactivated user, select **Activate**.
 
-* Hover over sections such as Manage Groups, Set up Templates, Access all data, 
-and Configure Facets to see a brief description of each permission.
-![Select Permissions](quick-start-images/select-user-permission.png)
+![Edit User Status](quick-start-images/11DeactivateUsers.png)
+***Edit user status.** To activate or deactivate a user, click on the three dots next to the user’s name and select either Activate or Deactivate. This change will take effect immediately.*
 
-* Assign specific permissions or all permissions to users.
-* Change passwords or deactivate users as needed.
+**2. Change User Password:**
+- To change a user's password, click on the three dots next to the user's name.
+- Select **Change Password**. A new window will appear, prompting you to authenticate yourself as an administrator.
+- After authenticating, enter and confirm the new password for the selected user. Ensure that the new password meets the required security criteria.
+- A confirmation window will notify you that the password has been successfully changed
 
-## Add, Update, and Delete Groups
+![User Password](quick-start-images/12ChangePasswords.png)
+***Change a user’s password**. Click on the menu next to the user's name, authenticate yourself as an admin (e.g. Genestack Superuser), and then set a new password for the user (e.g. New User 2024). Ensure the new password complies with security guidelines.*
 
-* To add, update, and delete groups **Manage groups** permission is required.
-* Click on Groups to explore existing groups and their members.
 
-![Create New Group](quick-start-images/add-new-group.png)
+As an administrator, your responsibilities include adding new users and managing their permissions, statuses, and passwords. For further details on managing permissions and groups, refer to the section Defining Permissions and User Groups in ODM.
 
-### Create new Groups
+## Add, update, and delete Groups
```

Comment:
Please, add here notification "To add, update, and delete groups **Manage groups** permission is required" as it was previously.